content/community/contributing/index.md

```diff
@@ -6,18 +6,33 @@ sidebar_multicard: false
 icon: tasks
 ---
  
-
-We are welcoming new contributors to the Keptn project!
+We welcome new contributors to the Keptn project,
+regardless of your background and experience!
 
 ## Getting started
 
+1. Join us on [Keptn Slack](/community/#slack).
+2. Join the `#help-contributing` channel.
+   It can be used for any questions related to Keptn contributions.
+   See the links to more specific channels, e.g. working groups, below.
+3. Explore the newcomer friendly issues, see the links below.
+
+## Keptn development
+
+If you are interested 
+The most of Keptn codebase is implemented in Golang and JavaScript/Angular.
+We also heavily use Kubernetes, Helm, Docker and
+other container technologies like CloudEvents or OpenTelemetry.
+
+### Keptn core components
+
 **1️.** If you are new to Keptn and want to get started contributing, please take a look at our [**CONTRIBUTING.md**](https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md) file that lays out the process on how to contribute to the project.
```

Comment:
```suggestion
**1️.** If you are new to Keptn and want to begin contributing, please take a look at our [**CONTRIBUTING.md**](https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md) file that lays out the process on how to contribute to the project.
```
I wonder if "If you are new to Keptn" is necessary -- I could be an experienced Keptn user and still be new to contributing, right?  I think the sentence holds up fine without this.

Also, shouldn't this also refer to the. https://github.com/keptn/keptn.github.io/blob/master/CONTRIBUTING.md file for people who want to contribute to the documentation?